nix/sources.json

```diff
@@ -36,27 +36,27 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "nixpkgs": {
-        "branch": "release-24.05",
+        "branch": "release-25.05",
```

Comment:
keep the `flake.nix` in sync with this?

Reply:
do you mean `nix flake update` or just branch name?

Reply:
I usually keep both branch name and rev the same, hopefully good for binary cache hits